### PR TITLE
[bugfix][feature] introduce fluentd_plugins_to_create

### PR DESCRIPTION
**Pull request by trombik**
also, fix a bug that causes a validation failure when creating `fluentd.conf`.

### CODE REVIEW

**Comment by trombik** [comment]
although  `fluentd_plugins_to_install`, and `fluentd_plugins_to_create` are not descriptive name, a project desperately needs the fix. they will be renamed, which implies major version bump.

**Comment by trombik** [comment]
issue #51 has been created

File: handlers/main.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@
   service:
     name: "{{ fluentd_service_name }}"
     state: restarted
-  notify: Wait 3 seconds (restart)
+  notify: Wait 5 seconds (restart)
 
-- name: Wait 3 seconds (restart)
+- name: Wait 5 seconds (restart)
```
Comment (mitsururike) [inline handlers/main.yml:10]: the comment below must be updated.
`# XXX listed order of these handlers does matter. "Wait 3 seconds (restart)"`

Comment (trombik) [reply handlers/main.yml:10]: fixed in 338363d

**Review by mitsururike — APPROVED** [review]

**Pull request merged**